src/components/Button/Button.jsx

```diff
@@ -53,7 +53,8 @@ class Button extends React.PureComponent {
 
     return React.Children.map(children, (child, index) => {
       if (child && !child.hasOwnProperty('type')) return child
-      if (child.type !== Icon) return child
+      if (!child || (child && child.type !== Icon)) return child
```

Comment:
Not a big deal, I find these a bit hard to read, perhaps...

```
if (!child) return
if (!child.hasOwnProperty('type')) return child
if (child.type !== Icon) return child
```

Maybe is just me though 😅

Reply:
that's a great suggestion. 🙇